src/RiddleFeedService.php

```diff
@@ -157,17 +165,58 @@ private function processRiddleResponse($riddleResponse) {
           continue;
         }
 
-        $feed[] = array(
-          'title' => $this->getRiddleTitle($riddleEntry),
-          'uid' => $riddleEntry['uid'],
-        );
+        $image = NULL;
+        if (!empty($riddleEntry['data']['image']['standard'])) {
```

Comment:
It would be better to move these "if" statements into ```getImage()``` function and then just execute ```'image' => $this->getImage($riddleEntry),```. So that logic is same as it's for ```getRiddleTitle()```.